learn/sherif/main_test/src/main/java/frc/robot/subsystems/IntakeControl.java

```diff
@@ -0,0 +1,34 @@
+package frc.robot.subsystems;
+
+import edu.wpi.first.wpilibj.PWMTalonSRX;
```

Comment:
This is the wrong Talon object. You want the WPI_TalonSRX object from the ctre phoenix library (`import com.ctre.phoenix.motorcontrol.can.WPI_TalonSRX;`)